radix-engine/src/track/track.rs

```diff
@@ -929,27 +965,23 @@ impl<'s, S: SubstateDatabase, M: DatabaseKeyMapper + 'static> CommitableSubstate
                     return Ok(items);
                 }
 
+                // TODO: Check that substate is not locked, before use outside of native blueprints
```

Comment:
This comment no longer applies to `track` since it doesn't manage locks. I believe there is a comment regarding this in `SubstateIO` or `CallFrame`

Reply:
I will give it a try :)